src/App.tsx

```diff
@@ -1,61 +1,36 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
-export const App = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <form action="/api/todos" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>Choose a user</option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { Todo } from './types/Todo';
+import { getNewTodoId } from './helpers/getNewTodoId';
 
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
-
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
+export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
 
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
+  const handleAddTodo = (todoInfo: {
+    title: string,
+    userId: number,
+  }) => {
```

Comment:
You can use some of the utility types, for example, `partial`, `pick`, or `omit`.
https://www.typescriptlang.org/docs/handbook/utility-types.html

`Pick<Todo, title | userId>`